### ISSUE TITLE
ADB Interface

### ISSUE THREAD

**Issue by Djuk1c** [Android1500/AndroidFaker#53]
Is there a way to randomize all selected parameters through adb?
So i dont have to open app and automate the clicks?

**Comment by Android1500** [comment]
> Is there a way to randomize all selected parameters through adb?
> 
> So i dont have to open app and automate the clicks?

We can think about that

**Comment by cr0un** [comment]
I agree, that would be handy. Now we have to use UI interface for this, it is certainly inconvenient for automation.
